test/property.test.js

```diff
@@ -62,37 +58,127 @@ describe('loopback:property generator', function() {
     });
   });
 
-  it('creates a typed array', function(done) {
+  it('should create model containing number type', function(done) {
     var propertyGenerator = givenPropertyGenerator();
     helpers.mockPrompt(propertyGenerator, {
       model: 'Car',
-      name: 'list',
+      name: 'age',
+      type: 'number',
+      required: 'true',
+      defaultValue: '555555555555555555555.5'
```

Comment:
yes, there's an explicit intention to ensure very large numbers are also handled